barretenberg/acir_tests/bbjs-test/src/index.ts

```diff
@@ -36,7 +36,7 @@ async function generateProof({
   const backend = new UltraHonkBackend(bytecode, { threads: multiThreaded ? 8 : 1 });
 
   const witness = await fs.readFile(witnessPath);
-  const proof = await backend.generateProof(new Uint8Array(witness), { keccak: (oracleHash === "keccak") });
+  const proof = await backend.generateProof(new Uint8Array(witness), { keccak: (oracleHash === "keccak"), starknet: (oracleHash === "starknet") });
```

Comment:
This seems like it's better as an enum now, adding two mutually exclusive options isn't ideal

Reply:
I though about it, but it would break the current typing of the API. Would that be ok?

Reply:
It's a good point, that's not a big breaking change but we can save it for a follow-up regardless. Not a blocker.